File: CS/CustomizedTrackBar/CustomTrackBar/CustomTrackBarViewInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraEditors.ViewInfo;
using DevExpress.XtraEditors.Repository;
using DevExpress.XtraEditors.Drawing;
using DevExpress.LookAndFeel;

namespace CustomizedTrackBar
{
    class CustomTrackBarViewInfo : TrackBarViewInfo
    {
        public CustomTrackBarViewInfo(RepositoryItem item) : base(item) { }

        public RepositoryItemCustomTrackBar RepositoryItem
        { get { return this.Item as RepositoryItemCustomTrackBar; } }

        public override TrackBarObjectPainter GetTrackPainter()
        {
            //if (IsPrinting)
            //    return new TrackBarObjectPainter();
            //if (this.LookAndFeel.ActiveStyle == ActiveLookAndFeelStyle.WindowsXP)
            //    return new WindowsXPTrackBarObjectPainter();
            //if (this.LookAndFeel.ActiveStyle == ActiveLookAndFeelStyle.Skin)
            return new SkinCustomTrackBarObjectPainter(LookAndFeel.ActiveLookAndFeel);
            //if (this.LookAndFeel.ActiveStyle == ActiveLookAndFeelStyle.Office2003)
            //    return new Office2003TrackBarObjectPainter();
            //return new TrackBarObjectPainter();
        }
    }
}

File: VB/CustomizedTrackBar/CustomTrackBar/CustomTrackBarViewInfo.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraEditors.ViewInfo
Imports DevExpress.XtraEditors.Repository
Imports DevExpress.XtraEditors.Drawing
Imports DevExpress.LookAndFeel

Namespace CustomizedTrackBar
	Friend Class CustomTrackBarViewInfo
		Inherits TrackBarViewInfo
		Public Sub New(ByVal item As RepositoryItem)
			MyBase.New(item)
		End Sub

		Public ReadOnly Property RepositoryItem() As RepositoryItemCustomTrackBar
			Get
				Return TryCast(Me.Item, RepositoryItemCustomTrackBar)
			End Get
		End Property

		Public Overrides Function GetTrackPainter() As TrackBarObjectPainter
			'if (IsPrinting)
			'    return new TrackBarObjectPainter();
			'if (this.LookAndFeel.ActiveStyle == ActiveLookAndFeelStyle.WindowsXP)
			'    return new WindowsXPTrackBarObjectPainter();
			'if (this.LookAndFeel.ActiveStyle == ActiveLookAndFeelStyle.Skin)
			Return New SkinCustomTrackBarObjectPainter(LookAndFeel.ActiveLookAndFeel)
			'if (this.LookAndFeel.ActiveStyle == ActiveLookAndFeelStyle.Office2003)
			'    return new Office2003TrackBarObjectPainter();
			'return new TrackBarObjectPainter();
		End Function
	End Class
End Namespace
